src/neural/blas/transforms.h

```diff
@@ -23,11 +23,11 @@
 
 namespace lczero {
```

Comment:
Delete the Transform constructor, so that nobody instanstiate by mistaque

Reply:
I don't see any?